app/modules/edit_content.js

```diff
@@ -8,8 +8,8 @@ export default function editTask() {
       if (e.key === 'Enter') {
         e.preventDefault();
         editedText = todo.innerText;
+        console.log(editedText);
       }
```

Comment:
- [ ] I see that you didn't add any new require changes since the last review 
All changes to the To-Do List should be saved in local storage. When I reload the page, the edited task is discarded. Consider ensuring that the edited task is persistent(saved in localstorage).
 please make sure to do that as the previous reviewer stated

Reply:
Sorry seems like I didn't commit my changes well. I'm fixing that.